package.json

```diff
@@ -14,6 +14,7 @@
     },
     "dependencies": {
         "@angular/animations": "10.2.1",
+        "@angular/youtube-player": "10.2.6",
```

Comment:
We already embed youtube videos for guided tours.
e.g. here https://artemis.ase.in.tum.de/#/courses/39/exercises (start tutorial)

We simply use an iframe for this: [`guided-tour.component.html:111`](https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/guided-tour/guided-tour.component.html#L111).
The URLs are defined here: [`guidedTourCourseExerciseOverview.json:11`](https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json#L11)

Reply:
But this produces the same error as the official youtube component which is just an abstraction above the iframe? 
![image](https://user-images.githubusercontent.com/29718932/97883411-1fd54c80-1d25-11eb-9f1b-b863050b1fb6.png)

Reply:
And I like the interactivity I gain from using the component for example stopping the video when the unit is closed.

Reply:
> Maybe also a local problem but the video is not shown in the student view
> ![image](https://user-images.githubusercontent.com/44401560/97882298-c4568f00-1d23-11eb-9dd4-c53664b7157a.png)
> As well as in the instructor view
> ![image](https://user-images.githubusercontent.com/44401560/97882377-d9332280-1d23-11eb-928f-5fb682c3bb60.png)
> Can someone confirm this?

![image](https://user-images.githubusercontent.com/29718932/97883984-d1747d80-1d25-11eb-879c-aa25325f1766.png)
You choose a video that can not be embedded

Reply:
So I have a version working without using the youtube i frame api and thus without using the angular youtube component. What is lost? It is not possible in any way to interact with the video, meaning it is not possible for example to pause the video when the unit is collapsed. Through a "hack" by reassigning the src attribute of the iframe you can make it that the video stops completely when the unit is collapsed. The problem with this is that the viewing progress is lost. There is also no feedback from the video anymore, thus no error messages can be displayed.

Reply:
I removed the YouTube Angular Component and YouTube I Frame APi in cda468f in favor of using a simple iframe approach. Some features are lost:

Videos now continue playing even though video unit is collapsed.
YouTube Errors are not received and not displayed in the Ui any more

Reply:
@krusche what do you think. Most importantly we should have a consistent way of embedding youtube videos throughout the platform.

Reply:
I think we use the easy version for now, we can still discuss improvements in a follow-up.
I would also favor consistency. Let's have a quick discussion tomorrow and check whether we should use the YouTube Angular Component for the other place as well.

By the way: not everyone is using Youtube, so we should also allow other videos to be embedded in lectures ;-)

Reply:
please remove the unused entry for now here